netcdf/attributes.go

```diff
@@ -40,6 +45,22 @@ func (a Attr) Len() (n uint64, err error) {
 	return
 }
 
+// ValueString returns the value of the attribute value of type text.
+func (a Attr) ValueString() (val string, err error) {
```

Comment:
Sorry, I meant Attr's ReadBytes methond, which actually does exist. Also, instead of using ReadBytes method directly, you can use GetBytes function:
```
func (a Attr) ValueString() (string, error) {
	b, err := GetBytes(a)
	if err != nil {
		return "", err
	}
	return string(b), nil
}
```